.changelog/get-dep-changes.sh

```
@@ -0,0 +1,382 @@
+    # Using a variable for the grep expression is a bit clunky, so I'm using fixed string here.
+    # Technically this will match both "<lib> =>" and "<pre><lib> =>". Matching the second one
+    # would be problematic, but it's not accounted for in here. The nature of the names of the
+    # libraries makes it highly unlikely to happen.
+    repl="$( grep -F "$lib =>" "$fn" | sed -E 's/^.* => //' )"
+    if [[ -n "$repl" ]]; then
+        if [[ "$repl" =~ ' ' ]]; then
+            # $repl has the format "<other library> <other version>"
+            # We'll want it as "<other version> of `<other library>`".
+            # It's provided without quotes to printf so that it gets split on that
+            # space and provided as two separate args to put tics around the <other library>.
+            printf '%s of `%s`' $( sed -E 's/^([^ ]+) +(.*)$/\2 \1/' <<< "$repl" )
```

Shellcheck (0.11.0)

[warning] This format string has 2 variables, but is passed 1 argument.

(SC2183)